src/auth.js

```diff
@@ -0,0 +1,54 @@
+import jwt from 'jsonwebtoken'
+
+export async function login(request, response) {
+    try {
+        //Get username
+        const {username} = request.body
+
+        //If no username this doesnt run
+        if(username) {
```

Comment:
If I'm reading this correctly, no authentication is required. Any username will do, and there is nothing checking for passwords.